test_schemas/en/test_routing_checkbox_count.json

```diff
@@ -0,0 +1,155 @@
+                                            },
+                                            2
+                                        ]
+                                    }
+                                },
+                                {
+                                    "block": "incorrect-answer"
+                                }
+                            ]
+                        },
+                        {
+                            "type": "Interstitial",
+                            "id": "incorrect-answer",
+                            "content": {
+                                "title": "You did not select 2 toppings",
+                                "contents": [
+                                    {
+                                        "description": {
+                                            "text": "You were asked to select <em>2</em> toppings but you actually selected <em>{answer}</em>.",
+                                            "placeholders": [
+                                                {
+                                                    "placeholder": "answer",
+                                                    "value": {
+                                                        "source": "answers",
+                                                        "identifier": "topping-checkbox-answer"
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    }
+                                ]
+                            },
+                            "routing_rules": [
+                                {
+                                    "section": "End"
+                                }
+                            ]
+                        },
+                        {
+                            "type": "Interstitial",
+                            "id": "correct-answer",
+                            "content": {
+                                "title": "Correct",
+                                "contents": [
+                                    {
+                                        "description": {
+                                            "text": "You were asked to select <em>2</em> toppings and you selected <em>{answer}</em>.",
+                                            "placeholders": [
+                                                {
+                                                    "placeholder": "answer",
+                                                    "value": {
+                                                        "source": "answers",
+                                                        "identifier": "topping-checkbox-answer"
+                                                    }
+                                                }
```

Comment:
Piping checkbox without any formatting looks odd, it outputs the Markup etc. Since we don't have count for placeholder, you can use `format_list` placeholder to make it look more appropriate.
```suggestion
                                                {
                                                    "placeholder": "answer",
                                                    "transforms": [
                                                        {
                                                            "transform": "format_list",
                                                            "arguments": {
                                                                "list_to_format": {
                                                                    "source": "answers",
                                                                    "identifier": "topping-checkbox-answer"
                                                                }
                                                            }
                                                        }
                                                    ]
                                                }
```

Reply:
Resolving because its a duplicate